extensions/amp-script/0.1/amp-script.js

```diff
@@ -43,7 +43,7 @@ export class AmpScript extends AMP.BaseElement {
   layoutCallback() {
     if (!isExperimentOn(this.win, 'amp-script')) {
       const error = 'Experiment "amp-script" is not enabled.';
-      user().error(TAG, error);
```

Comment:
The above const is not necessary now?

Reply:
It is used one line below lol, but yes you're right it's also just a string. I will make that change.